src/Integration/Laravel/Cache/src/ServiceProvider.php

```diff
@@ -15,20 +15,51 @@ public function boot()
         /** @var CacheManager $manager */
         $manager = $this->app['cache'];
 
-        $manager->extend('async-aws-dynamodb', \Closure::fromCallable([$this, 'createStore']));
+        $closure = $this->getClosure();
```

Comment:
Missing redis implementation

Reply:
One can use `Illuminate\Cache\RedisStore` since it is not related to the AWS SDK